rakefile

```diff
@@ -106,7 +178,7 @@ rototiller_task :performance do |t|
       option.name = '--preserve-hosts'
       option.message = 'Whether to preserve hosts or not'
       option.add_argument do |arg|
-        arg.name = 'always'
+        arg.name = 'never'
```

Comment:
this is probably useful outside of this commit.  should be separated.  e.g. if the other code needs reverting, we'd want this to stay.  correct?

Reply:
Actually, I need to update my PR, apparently this is not going to be respected either way with the new ABS workflow, so we'll need to do our own cleanup with the new de-provision task.